cross/c-ares/Makefile

```diff
@@ -2,7 +2,8 @@ PKG_NAME = c-ares
 PKG_VERS = 1.27.0
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = https://c-ares.org/download
+PKG_DIST_DIR = $(subst ., _ ,$(subst -,,$(PKG_NAME))-$(PKG_VERS))
+PKG_DIST_SITE = https://github.com/c-ares/$(PKG_NAME)/releases/download/${PKG_DIST_DIR}
```

Comment:
If you need to touch this makefile, it is probably worth an update to the latest version (1.29)

and please keep the code simple (and readable) like
`PKG_DIST_DIR = cares-$(subst ., _ ,$(PKG_VERS))`

_OR_ in one line:
`PKG_DIST_SITE = https://github.com/c-ares/c-ares/releases/download/cares-$(subst ., _ ,$(PKG_VERS))`